tasks/ssl_certificate.yml

```diff
@@ -0,0 +1,32 @@
+---
+- include_vars: "group_vars/env-production/vault.yml"
+
+- name: create admin.juwai.com + digicert certificate file
+  template:
+    src: "admin.juwai.com.crt.j2"
+    dest: "/etc/ssl/admin.juwai.com.crt"
+    owner: root
+    group: root
+    mode: 0600
+
+- name: create admin.juwai.com key file
+  copy:
+    content: "{{ admin_juwai_com_key }}"
+    dest: "/etc/ssl/admin.juwai.com.key"
+    owner: root
+    group: '{{ web_server_user }}'
+    mode: 0640
```

Comment:
We only want root to able to read this file. 640 is only for a special case where PHP needs to read the file too.